pubsub/drivertest/drivertest.go

```diff
@@ -150,6 +159,13 @@ func RunConformanceTests(t *testing.T, newHarness HarnessMaker, asTests []AsTest
 		"TestMetadata":                                            testMetadata,
 		"TestNonUTF8MessageBody":                                  testNonUTF8MessageBody,
 	}
+	if opts.SkipTestsOfNonExistentThings {
```

Comment:
It would be more clear to have the tests be added on `!opts.SkipTestsOfNonExistentThings`, instead of pattern matching on the name.

Reply:
Done.